apps/condo/domains/common/components/LayoutContext.tsx

```diff
@@ -16,6 +15,13 @@ interface ILayoutContext {
     addNotification?: (notification: ITopNotification) => void
 }
 
+const detectMobileNavigator = (): boolean => {
+    return (
+        typeof window !== 'undefined'
+        && /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(window.navigator.userAgent)
```

Comment:
for optimization sake it's better to move this static RegExp to a constant, in order to skip extra compilations in runtime